ui/job-view/details/summary/SummaryPanel.jsx

```diff
@@ -18,18 +18,19 @@ export default class SummaryPanel extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!nextProps.selectedJob || !Object.keys(nextProps.selectedJob).length) {
+    const { selectedJob } = nextProps;
+    if (!selectedJob || !Object.keys(selectedJob).length) {
       return;
     }
 
-    this.setJobMachineUrl(nextProps);
+    this.setJobMachineUrl(selectedJob);
   }
 
-  async setJobMachineUrl(props) {
+  async setJobMachineUrl(job) {
     let machineUrl = null;
 
     try {
-      machineUrl = await getJobMachineUrl(props);
+      machineUrl = await getJobMachineUrl(job);
     } catch (err) {
       machineUrl = '';
```

Comment:
@camd, should we change this so exceptions are not hidden? Also, if the URL is the empty string, it currently results in the link that causes a full page reload - perhaps `'#'` or `null` would be better?

Reply:
Hmm...  Another good spot.  I'm not sure what to do with the exception though.  I suppose we could use a ``console.error`` in that case?  But you make a great point.  I think it should be set to ``null`` if we don't get it, and in the ``render`` we should display it as just text instead of a link.